### PR TITLE
data structures:update vote

### PR DESCRIPTION
**Pull request by marbar3778**
## Description

update vote

I have linting enabled locally, let me know if youd like me to revert.

### CODE REVIEW

**Comment by adlerjohn** [comment]
Now that #137 is in, can you rebase against `master` @marbar3778 (which should elide your linting changes in this PR)?

**Comment by marbar3778** [comment]
done

**Review by adlerjohn — DISMISSED** [review]
Sooo much better without the linting noise 😂

File: specs/data_structures.md [hunk 1]
```diff
@@ -196,6 +196,8 @@ Address is a [type alias](#type-aliases).
 
 Addresses are the [hash](#hashing) [digest](#hashdigest) of the [public key](#publickey).
 
+Addresses are a length of 32.
```
Comment (liamsi) [inline specs/data_structures.md:199]:
```suggestion
Addresses have a length of 32 bytes.
```

**Review by liamsi — DISMISSED** [review]
minor optional nit

**Review by adlerjohn — APPROVED** [review]

**Pull request merged**